### PR TITLE
Update privacy notice on coronawarn.app for version 1.13

### PR DESCRIPTION
**Pull request by dsarkar**
[EXPOSUREAPP-5332](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-5332)

### CODE REVIEW

**Comment by MikeMcC399** [comment]
Will 1.13.0 Android or 1.13.2 be released?
(See https://github.com/corona-warn-app/cwa-app-android/releases/tag/v1.13.2-RC0)

**Comment by dsarkar** [comment]
@MikeMcC399 I am not sure yet. I will adapt the gulpfile last minute.

File: gulpfile.js [hunk 1]
```diff
@@ -314,10 +314,10 @@ function replaceVersionNumbers() {
     .src([PATHS.dist + "/**/*.html"])
     .pipe(replace('[ios.latest-os-version]', '14.4'))
     .pipe(replace('[ios.minimum-required-os-version]', '12.5'))
-    .pipe(replace('[ios.current-app-version]', '1.12.1'))
+    .pipe(replace('[ios.current-app-version]', '1.13.0'))
     .pipe(replace('[android.latest-os-version]', '11'))
     .pipe(replace('[android.minimum-required-os-version]', '6'))
-    .pipe(replace('[android.current-app-version]', '1.12'))
+    .pipe(replace('[android.current-app-version]', '1.13.0'))
```
Comment (Ein-Tim) [inline gulpfile.js:320]:
```suggestion
    .pipe(replace('[android.current-app-version]', '1.13.2'))
```

Should be 1.13.2 (see https://github.com/corona-warn-app/cwa-app-android/releases/tag/v1.13.2)

**Comment by Ein-Tim** [comment]
@dsarkar 

Should this be merged ASAP with 1.13 released?

**Comment by dsarkar** [comment]
@Ein-Tim I will merge this one now. Could you create a PR with an update of the Android CWA version number? Thanks !

**Comment by Ein-Tim** [comment]
@dsarkar 

For sure 👍

**Comment by Ein-Tim** [comment]
@dsarkar 
#940

**Pull request merged**